src/ducks/balance/components/GroupPanel.spec.jsx

```diff
@@ -116,9 +116,16 @@ describe('GroupPanel', () => {
 })
 
 describe('Reimbursement virtual group styling', () => {
+  // We need a different date for each test otherwise selectors are not
+  // recomputed since queryCreateSelector bases its memoization on
+  // the lastUpdate of the query
+  const mockedDate = +new Date('2020-01-31T00:00')
+  let i = 0
   beforeEach(() => {
-    MockDate.set('2020-01-31T00:00')
+    MockDate.set(mockedDate + i++)
```

Comment:
It seems that we will have to add this everytime we have test with cozy-client data and mocked date? I'm afraid we will forgot it :(

Reply:
> I'm afraid we will forgot it :(

The test cannot work if we do not do this so I don't think we will forget it, I do not see yet a better solution, I will speak about this in Front meeting.

Reply:
Okay